src/my-account/controllers/my-account.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Get, Patch, Delete } from '@nestjs/common';
+import { MyAccountService } from '../services/my-account.service';
+import { ApiTags } from '@nestjs/swagger';
+
+@Controller('me')
```

Comment:
I prefer to change name of the Controller from `me` to `my-account` or `account`